src/react-contenteditable.js

```diff
@@ -7,20 +7,18 @@ export default class ContentEditable extends React.Component {
   }
 
   render() {
-    var props = {};
-    Object.assign(props, this.props);
-    delete props.tagName;
-    delete props.html;
```

Comment:
I don't get your point.
Could you explain your question more explicitly? Thanks!